src/appkit.cs

```diff
@@ -26692,4 +26729,338 @@ interface NSWindowTabGroup
+		NSObject Target { get; set; }
+
+		[NullAllowed, Export ("action", ArgumentSemantic.Assign)]
+		Selector Action { get; set; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Export ("customizationLabel")]
+		string CustomizationLabel { get; set; }
+	}
+
+	[Mac (10,15)]
+	[Native]
+	public enum NSCollectionLayoutSectionOrthogonalScrollingBehavior : long
+	{
+		None,
+		Continuous,
+		ContinuousGroupLeadingBoundary,
+		Paging,
+		GroupPaging,
+		GroupPagingCentered,
+	}
+
+	[Flags, Mac (10,15)]
+	[Native]
+	public enum NSDirectionalRectEdge : ulong
+	{
+		None = 0x0,
+		Top = 1uL << 0,
+		Leading = 1uL << 1,
+		Bottom = 1uL << 2,
+		Trailing = 1uL << 3,
+		All = Top | Leading | Bottom | Trailing,
+	}
+
+	[Mac (10,15)]
+	[Native]
+	public enum NSPickerTouchBarItemControlRepresentation : long
+	{
+		Automatic = 0,
+		Expanded = 1,
+		Collapsed = 2,
+	}
+
+	[Mac (10,15)]
+	[Native]
+	public enum NSPickerTouchBarItemSelectionMode : long
+	{
+		SelectOne = 0,
+		SelectAny = 1,
+		Momentary = 2,
+	}
+
+	[Mac (10,15)]
+	[Native]
+	public enum NSRectAlignment : long
+	{
+		None = 0,
+		Top,
+		TopLeading,
+		Leading,
+		BottomLeading,
+		Bottom,
+		BottomTrailing,
+		Trailing,
+		TopTrailing,
+	}
+
+	[Native]
+	public enum NSTextInputTraitType : long
+	{
+		Default,
+		No,
+		Yes,
+	}
+
+	[Mac (10,15)]
+	[Native]
+	public enum NSToolbarItemGroupControlRepresentation : long
+	{
+		Automatic,
+		Expanded,
+		Collapsed,
+	}
+
+	[Mac (10,15), iOS (13,0)]
+	[Native]
+	public enum NSToolbarItemGroupSelectionMode : long
+	{
+		SelectOne = 0,
+		SelectAny = 1,
+		Momentary = 2,
+	}
+
+	[Mac (10,15)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSCollectionLayoutAnchor : NSCopying, INSCopying
+	{
+		[Static]
+		[Export ("layoutAnchorWithEdges:")]
+		NSCollectionLayoutAnchor LayoutAnchorWithEdges (NSDirectionalRectEdge edges);
+
+		[Static]
+		[Export ("layoutAnchorWithEdges:absoluteOffset:")]
+		NSCollectionLayoutAnchor LayoutAnchorWithEdgesAbsoluteOffset (NSDirectionalRectEdge edges, CGPoint absoluteOffset);
+
+		[Static]
+		[Export ("layoutAnchorWithEdges:fractionalOffset:")]
+		NSCollectionLayoutAnchor LayoutAnchorWithEdgesFractionalOffset (NSDirectionalRectEdge edges, CGPoint fractionalOffset);
+
+		[Export ("edges")]
+		NSDirectionalRectEdge Edges { get; }
+
+		[Export ("offset")]
+		CGPoint Offset { get; }
+
+		[Export ("isAbsoluteOffset")]
+		bool IsAbsoluteOffset { get; }
+
+		[Export ("isFractionalOffset")]
+		bool IsFractionalOffset { get; }
+	}
+
+	[Mac (10,15)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSCollectionLayoutDimension : NSCopying, INSCopying
+	{
+		[Static]
+		[Export ("fractionalWidthDimension:")]
+		NSCollectionLayoutDimension CreateFractionalWidthDimension (nfloat fractionalWidth);
+
+		[Static]
+		[Export ("fractionalHeightDimension:")]
+		NSCollectionLayoutDimension CreateFractionalHeightDimension (nfloat fractionalHeight);
```

Comment:
Remove with dimension. Following the comment above, enum usage and a single method?